workers/genotype_extractor.py

```diff
@@ -21,6 +21,11 @@
 
 @worker.task(bind=True)
 def extract(self, vcf_id):
+    register_running_task(self, vcf_id)
+    return _extract(vcf_id)
```

Comment:
It registered the task with the database; it required that we run the worker queue, so I don't want to test that.